modules/telco-core-about-the-telco-core-cluster-use-model.adoc

```diff
@@ -6,20 +6,19 @@
 [id="telco-core-about-the-telco-core-cluster-use-model_{context}"]
 = About the telco core cluster use model
 
-The telco core cluster use model is designed for clusters that run on commodity hardware.
-Telco core clusters support large scale telco applications including control plane functions like signaling, aggregation, session border controller (SBC), and centralized data plane functions such as 5G user plane functions (UPF).
+The telco core cluster use model is designed for clusters running on commodity hardware.
+Telco core clusters support large scale telco applications including control plane functions like signaling, aggregation, session border controller (SBC); and centralized data plane functions such as 5G user plane functions (UPF).
```

Comment:
I think the oxford comma is ok here? It doesn't separate an independent clause, the other list items use commas.